src/server/pegasus_write_service_impl.h

```diff
@@ -612,9 +613,16 @@ class pegasus_write_service::impl : public dsn::replication::replica_base
 
         FAIL_POINT_INJECT_F("db_write", [](dsn::string_view) -> int { return FAIL_DB_WRITE; });
 
+        rocksdb::Status status =
+            _batch.Put(_meta_cf, pegasus_server_impl::LAST_FLUSHED_DECREE, std::to_string(decree));
+        if (dsn_unlikely(!status.ok())) {
+            derror_rocksdb("Write", status.ToString(), "decree: {}", decree);
```

Comment:
This log is same as write actual data failed log, could you please add more description to show actual error?